Wiki/Programs/PokemonSwSh/BoxReorderNationalDex.md

```diff
@@ -0,0 +1,44 @@
+# Box Reorder National Dex
+
+## Program Description
+
+Order boxes of Pokémon, based on their national dex index.
```

Comment:
Image Kudasai? :)

Reply:
image added ! let me know if this one is good or not !